### ISSUE TITLE
Provide a thicken method for Shell

### ISSUE THREAD

**Issue by lorenzncode** [CadQuery/cadquery#1269]
```py
import cadquery as cq

lines = cq.Workplane().lineTo(10, 0).lineTo(20, 4)
lines = lines.consolidateWires()

path = cq.Workplane("XZ").lineTo(0, 10)
result_sweep = lines.sweep(path)

# Shell: result.sweep.val()
# thicken not supported here
result_thicken = result_sweep.val().thicken(0.2)
result_thicken = result_thicken.clean()

show_object(result_thicken)
```

**Comment by adam-urbanczyk** [comment]
I guess you want to move `Face.thicken` to a hypothetical `Mixin2D` class?

**Comment by lorenzncode** [comment]
Yes a new `Mixin2D` class  sounds good.

**Comment by adam-urbanczyk** [comment]
Implemented in #1469, see `offset`
